tempodb/backend/s3/s3.go

```diff
@@ -386,6 +386,8 @@ func createCore(cfg *Config, hedge bool) (*minio.Core, error) {
 
 	if cfg.ForcePathStyle {
 		opts.BucketLookup = minio.BucketLookupPath
+	} else {
```

Comment:
If both options can be false, then perhaps we want an `else if` here.  Are these options mutually exclusive?

Reply:
minio.BucketLookup default value is  Auto(Zero)，RequestPathStyle is optional：0,1,2.

Reply:
ForcePathStyle Highest priority.